CHANGELOG.rst

```diff
@@ -2,6 +2,22 @@
 Changelog
 =========
 
+0.19.0 (YYYY-MM-DD)
+===================
+
+* Now requires ``attrs >= 22.2.0``.
+
+* Added the :class:`~.DropLowProbabilityItemPipeline` that drops items with the
+  ``probability`` value lower than a set threshold.
+
+* Added the :attr:`~.ListMetadata.validationMessages` attribute.
+
+* Reorganized the module structure.
```

Comment:
Should this be more verbose?

Reply:
Since most (all?) changes are internal, this is probably OK.

Reply:
There are also some classes which are promoted to public, e.g. BaseMetadata, ListMetadata, DetailsMetadata

Reply:
We may also mention zyte_common_items.converters module; the code was scattered in various places before, but now it contains useful attrs convertors in a single place

Reply:
Just checked it; it seems that's only these 3 classes :)